packages/experiments-realm/app-task-card.gts

```diff
@@ -261,6 +324,33 @@ class AppTaskCardIsolated extends Component<typeof AppTaskCard> {
         display: inline-block;
       }
     </style>
+    <style scoped>
```

Comment:
There are two scoped-style blocks currently, would it be simpler if we merge into one?